### ISSUE TITLE
Past Papers and Topic Notes view

### ISSUE THREAD

**Issue by MaskyS** [MaskyS/Studento#30]
Zero of all, we need to compile all the past papers and topic notes we are going to use. Then, all the past papers need to be converted to html (by using either pdf2html or an online service) and also optimise all assets and the html file itself. We need to do this because currently there are no pdf viewing plugin available for flutter.

Next, all of the html files need to edited. let's say that our html uses an image. Previously the path was as such: 
```<img src ="mathematics_paper_1_2016/img1.jpg"/>```

Now it needs to be:

```<img src="/assets/past-papers/assets/{syllabus_code}/mathematics_paper_1_2016/img1.jpg"/>```

Past papers need to be downloaded from our Firebase Storage, this is tracked in another issue: #31  
When these are downloaded, these should go inside ```/assets/{past_papers or topic_notes}```

The asset directory should look like 

```--/assets
--/assets/past_papers/{o_level or a level}
--/--------/-----------------/mathematics/
--/--------/-----------------/-----------------/mathematics_paper_1_2016/
--/--------/-----------------/-----------------/mathematics_paper_1_2016/index.html
--/--------/-----------------/-----------------/mathematics_paper_1_2016/assets/
--/--------/-----------------/-----------------/mathematics_paper_1_2016/assets/img1.jpg
--/assets/topic_notes
// same as above^
```

We need a script that adds either adds the new files to pubspec.yaml or uncomments specific lines in the file. This should only be done once first-time setup is complete and files have been downloaded (Option 1 is probably cleaner though harder). This is tracked in issue: #38 

We need to implement an algorithm that gets us the filename of the html that needs to be displayed. This is tracked in issue #33

In past_papers_view and topic_list_view, we need to start up a local server using Jaguar, the root directory should be selected based on whether a past papers or topic notes file is requested. This is tracked in issue: #34

**Comment by MaskyS** [comment]
This issue is now irrelevant since we have decided to move to PDFs instead of html files.
